raster/r.in.poly/r.in.poly.html

```diff
@@ -31,7 +31,7 @@ <h2>NOTES</h2>
 <p>
 Polygons are filled, i.e. they define an area.
 
-<h3 id="format">Input Format</h3>
+<h3>Input Format</h3>
```

Comment:
The id is generated back on rendering right? (There's nothing to change here, just observing)

Reply:
Yes, right.